.github/workflows/build.yml

```diff
@@ -0,0 +1,76 @@
+name: Build
+on:
+    push:
+        branches:
+            - master
+    pull_request:
+        branches:
+            - master
+jobs:
+    build:
+        name: PHP ${{ matrix.php-versions }} ${{ matrix.composer-args }}
+        strategy:
+            fail-fast: false
+            matrix:
+                operating-system: [ubuntu-latest]
+                php-versions: ['7.2', '7.3', '7.4']
+                composer-args: ['', '--prefer-lowest']
+                allow-failure: [false]
+                include:
+                    - operating-system: ubuntu-latest
```

Comment:
Because PHP8 needs special config to bypass the platform checks in composer, I had to add a couple of include sections here.  Eventually, once PHP8 is stable and all the packages we install allow it, we can remove these and just specify `8.0` in the `php-versions` array above.

Reply:
Curious if this is still necessary, as it's been a while since this PR was opened?

Reply:
I was able to remove the `--no-scripts` flag from the regular PHP 8 run, but the `prefer-lowest` run still needs that flag.

It isn't possible to remove the `ignore-platform-reqs` flag just yet as `composer-require-checker` doesn't have a version that allows PHP 8 yet.